src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/Project/ProjectDetailTabs.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { Tab, Tabs } from '@mui/material';
+import { useEffect, useMemo, useState } from 'react';
+import { Link, useLocation, useRouteMatch } from 'react-router-dom';
```

Comment:
we rename all links from react-router-dom to RouterLink because it conflicts with the mui links

Reply:
done